src/App.js

```diff
@@ -1,11 +1,19 @@
 import React from "react";
-
+import { BrowserRouter as Router, Link, Route, Routes } from "react-router-dom";
 import "styles/main.scss";
```

Comment:
we are following a proper way of importing like first comes imports form libraries then comes our component imports then comes any utility imports then any asset imports and at last css imports

Reply:
resolved